requirements.txt

```diff
@@ -1,25 +1,71 @@
-certifi==2021.5.30        # via requests
-charset-normalizer==2.0.6  # via requests
-click==8.0.1              # via flask
-flask-jwt-extended==3.24.1  # via patientsearch (setup.py)
+#
+# This file is autogenerated by pip-compile with Python 3.9
+# by the following command:
+#
+#    pip-compile
+#
+async-timeout==4.0.3
+    # via redis
+blinker==1.8.2
+    # via flask
+cachelib==0.13.0
+    # via flask-session
+certifi==2024.7.4
+    # via requests
+charset-normalizer==3.3.2
+    # via requests
+click==8.1.7
+    # via flask
+flask==2.0.1
```

Comment:
it looks like you had to downgrade some dependencies after running `pip-compile`

Could you add a pin (and comment) in setup.cfg for each dependency downgraded, so it doesn't get overwritten next time pip-compile is run?